opm/simulators/wells/WellState.hpp

```diff
@@ -180,6 +180,11 @@ class WellState
         return this->global_well_info.value().is_open(name);
     }
 
+    bool getGlobalEfficiencyScalingFactor(const std::string& name) const
```

Comment:
took me a while to track this one. not a bool!

Reply:
Wow, the joys of copy-paste to save a few keystrokes.. Thanks!